_test/test/goldens/generated_build_script.dart

```diff
@@ -7,6 +7,7 @@ import 'package:build_web_compilers/builders.dart' as _i6;
 import 'package:build/build.dart' as _i7;
 import 'package:build_vm_compilers/builders.dart' as _i8;
 import 'dart:isolate' as _i9;
+import 'package:build_runner/build_runner.dart' as _i10;
```

Comment:
is it possible to move to other package imports?

Reply:
This is used for the `run` method, which is now the only api from the build_runner package